filament/backend/src/opengl/OpenGLDriver.h

```diff
@@ -83,11 +83,9 @@ class OpenGLDriver final : public DriverBase {
         }
 
         struct {
+            GLuint id;
```

Comment:
That's very unfortunate, it'll double the size of this structure for the regular case. I can't think of a better way though.